static/js/components/dashboard/courses/Grades_test.js

```diff
@@ -123,4 +129,10 @@ describe('Course Grades', () => {
     course.has_exam = false;
     assert.equal(renderGrades().find('.passed-course').length, 0);
   });
+
+  it('should call setShowGradeDetailDialog onClick', () => {
```

Comment:
I added a test that calls the `onRequestClose` prop on the dialog - I had trouble getting a click or keypress based test to work correctly.